### PR TITLE
[CHE:264]: Add Liveness check to the collector

### PR DESCRIPTION
**Pull request by bswetalina**
Story : https://cloudhealthtech.atlassian.net/browse/CHE-264

### CODE REVIEW

File: cloudhealth-collector/values.yaml [hunk 1]
```diff
@@ -81,7 +86,7 @@ rbac:
   create: true
 
 devArgs: |
-  ['upload_k8s_state_v3', '--verbose']
+  ['upload_k8s_state_v4', '--verbose']
```
[the remark this answers is not in the document]
Comment (bswetalina) [reply cloudhealth-collector/values.yaml]: fixed the typo left from testing

**Review by wxia20 — APPROVED** [review]

**Review by trjaskoviak — APPROVED** [review]

File: cloudhealth-collector/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kscherme) [reply cloudhealth-collector/values.yaml]: Was the zip file regenerated after this fix?

**Review by kscherme — COMMENTED** [review]
Following up on Pavan's questions from the Jira ticket: 
```
A couple of questions.
1. Does a customer have to re-deploy the agent using Helm or deployment.yaml to get access to the new probe?
2. Does updating work fine?
```

Has updating been tested?

File: cloudhealth-collector/values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (bswetalina) [reply cloudhealth-collector/values.yaml]: Regenerated the zip file after the fix

**Review by kscherme — APPROVED** [review]

**Review by gm-cht — APPROVED** [review]

**Pull request merged**